pkgs/top-level/all-packages.nix

```diff
@@ -1255,7 +1255,9 @@ with pkgs;
 
   arduino-cli = callPackage ../development/embedded/arduino/arduino-cli { };
 
-  arduino-core = callPackage ../development/embedded/arduino/arduino-core { };
+  arduinoPackages = dontRecurseIntoAttrs (callPackage ../development/embedded/arduino/arduino-core { });
```

Comment:
Do you even need this extra scope at this point?

Reply:
i do not know for sure. I copied it from the `platformio`. Should i remove it?

Reply:
Yes, that should reduce the diff of this PR.

Reply:
Still there?

Reply:
no removed

Reply:
Can see it on github and this comment is not marked with "outdated"

Reply:
:open_mouth:  in the "Conversations" it does show up, but in "Files changed"/"Commits" it does not show up.

Reply:
i force pushed all again. no difference. No idea what is going on with GitHub here...

Reply:
I'm not sure what is the confusion. I do:
```
# git clone git@github.com:Stunkymonkey/nixpkgs.git -b arduino-buildFHSUserEnv
# grep arduinoPackages nixpkgs/pkgs/top-level/all-packages.nix
  arduinoPackages = callPackage ../development/embedded/arduino/arduino-core { };
  arduino-core = arduinoPackages.arduino-chrootenv;
  arduino-unwrapped = arduinoPackages.arduino-unwrapped;
```

Reply:
i am talking about `dontRecurseIntoAttrs`, which is no longer present. What is your suggested change?

Reply:
something like this?

Reply:
I was talking about dropping the use of the `newScope`

Reply:
sorry for the misunderstanding.
It does not compile without. Either I am doing it wrong, or it is needed.